bedrock/firefox/templates/firefox/channel/ios.html

```diff
@@ -9,8 +9,6 @@
 {% block page_title %}
 {% if l10n_has_tag('firefox_title_meta_bug1345338') %}
   {{ _('Try New Features in a Pre-Release iOS Browser | Firefox') }}
-{% else %}
-  {{_('Test beta versions of Firefox for iOS via Apple’s TestFlight program and help make our mobile browser for iPhone, iPad and iPod touch even better.')}}
```

Comment:
Looking at the desktop and android pages, they both have an else with the old string. We could do the same here (it looks like all these pages share the same index.lang file?) or update the other two templates to remove the else and use the same base fallback... either approach works but we should be consistent.

Sorry what seemed to be a simple fix has turned so messy :/